airbyte-integrations/connectors/source-klaviyo/unit_tests/test_source.py

```diff
@@ -35,3 +36,11 @@ def test_check_connection(requests_mock, status_code, response, is_connection_su
     success, error = source.check_connection(logger=None, config={"api_key": "api_key"})
     assert success is is_connection_successful
     assert error == error_msg
+
+
+def test_streams():
+    source = SourceKlaviyo()
+    config = {"api_key": "some_key", "start_date": pendulum.datetime(2020, 10, 10).isoformat()}
+    streams = source.streams(config)
+    expected_streams_number = 8
```

Comment:
for good measure can we add an assertion that all of the stream names are unique?

Reply:
Updated, thanks for the advice!